timeless/items/views.py

```diff
@@ -11,11 +12,23 @@
 BP = Blueprint("items", __name__, url_prefix="/items")
 
 
-@BP.route("/")
-def list_items():
-    """ List the items """
-    items = [{"id": 1}, {"id": 2}]
-    return render_template("items/list.html", items=items)
+class ItemListView(ListView):
```

Comment:
@paulodamaso Seems it's wrong way. 
What just you need:

```
class ItemListView(ListView):
    "List all Items"
    template_name = "items/list.html"
    model = Item


List.register(bp, "/")
```

Reply:
@vladarefiev Actually I'm trying to untie the database model and logic of the view so we could unit test it. For this I have to inject `model` and `models`, where we will inject the mock or the database model

Reply:
@vladarefiev I've done the way you told but now we're getting an `301`, take a look

Reply:
@paulodamaso I didn't get your point about redirect, where should we do redirect?